include/javascript/mozaik/jquery.mozaik.js

```diff
@@ -46,7 +46,7 @@ var mozaik = {
     getMozaikInnerHTML: function(name, innertext, style) {
         //style = ($(innertext).attr('style') ? $(innertext).attr('style') + ';' : '') + style;
         //style = style.replace(/;\s*;/, ';');
-        var html = '<div class="mozaik-inner"' + (name ? ' data-name="' + name + '"' : '') + ' style="' + style + '">' + innertext + '</div>';
+        var html = '<div class="mozaik-inner"' + (name ? ' data-name="' + name + '"' : '') + ' style="background-color: #fafafa; ' + style + '">' + innertext + '</div>';
```

Comment:
Definitely not. If you add and apply the background-color only in the css it's looks like the color on the template block in the editor area because the browser render the colour on it (as normally) but it's never get inline on the element and it doesn't saved to the database so later when you will send the emails without the background colour information then it never send out via email contents and the recipients doesn't see the changed default background colour.